### ISSUE TITLE
Last.fm known issues

### ISSUE THREAD

**Issue by 19379** [19379/wsh_marc2003#3]
The following features are not working because of problems with the Last.fm API. I have absolutely no idea when/if they are going to be fixed.

artist top albums
artist top fans
user recommendations

The following are working at the time of this post:

similar artists
artist top tracks
artist top tags
all chart types
artist bio

**Comment by arcos26** [comment]
Hi marc2003, and thanks for all your script, wonderful !!!
My foobar without Your "now playing"script is really sad...do you have the project to update it ?

**Comment by 19379** [comment]
No, I will not be providing that script. From the **Updates & Support** page on the wiki...

> I'm not taking feature requests and I'm not restoring any scripts or features missing from previous packages.

**Comment by arcos26** [comment]
oh ...it's a bad news for me.
